keras_cv/layers/object_detection/non_max_suppression.py

```diff
@@ -121,12 +121,18 @@ def call(self, predictions, images=None):
             self.confidence_threshold,
             clip_boxes=False,
         )
-
         # output will be a ragged tensor because num_boxes will change across the batch
-        return self._encode_to_ragged(nmsed_boxes, images)
+        boxes = self._encode_to_dense_output_tensor(nmsed_boxes)
```

Comment:
Maybe this can have a different name? This function combines the boxes with the classes/scores and appends all the bounding boxes to a TensorArray.

This is fine too though.

Reply:
thanks for the feedback, I picked => _decode_nms_boxes_to_tensor